src/client/components/UserPage/AnnouncementModal/index.tsx

```diff
@@ -210,7 +230,7 @@ export default function AnnouncementModal() {
             variant="text"
             className={classes.learnMoreButton}
           >
-            Learn More
+            Try it now
```

Comment:
Hang on, didn't Lisa say it was okay to leave the button text?

Reply:
we decided to change the button text specially for this release as 'try it now' refers to an action